oxidauth-postgres/migrations/20240416170202_create_email_codes.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE email_codes (
+    id UUID PRIMARY KEY NOT NULL DEFAULT uuid_generate_v4(),
+    user_id UUID NOT NULL,
+    code VARCHAR(256) UNIQUE NOT NULL,
+    expires_at TIMESTAMPTZ NOT NULL,
+    created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+);
+
+CREATE INDEX email_code_user_id ON email_codes(user_id);
```

Comment:
Not sure the index is really necessary, but I assume we will accumulate codes quickly so this might be good